docs/DEVELOPER_GUIDE.md

```diff
@@ -108,20 +108,24 @@ spec:
       - {key: os, operator: In, values: [darwin, linux]}
     # url for downloading the package archive:
     uri: https://github.com/example/foo/releases/v1.0.zip
-    # sha256sum of the file downloaded above:
+    # sha256sum of the above archive file:
     sha256: "208fde0b9f42ef71f79864b1ce594a70832c47dc5426e10ca73bf02e54d499d0"
     # copy the used files out of the zip archive, defaults to `[{from: "*", to: "."}]`
     files:
     - from: "/foo-*/unix/*.sh" # path to the files extracted from archive
-      to: "."               # '.' refers to the root of plugin install directory
+      to:  "."                 # '.' refers to the root of plugin install directory
+    - from: "/foo-*/LICENSE"   # always install your LICENSE file
```

Comment:
At some point it would be better if we call these without / prefix. No prefix or ./ is better for sure.

Reply:
Hooray! Thanks for saying that. This annoyed me a lot as well.